Documentation/running-lkvm-stage1.md

```diff
@@ -77,6 +77,8 @@ If you want to see the kernel and boot messages, run rkt with the `--debug` flag
 
 You can exit pressing `<Ctrl-a x>`.
 
+Notes: By default containers start working on all cpus and with 1GB. If user specify parameters via pod-manifest, memory will be increased by 128MB. Additional memory is required by system process.
```

Comment:
System process? Which?
"by system" is enough.

If user specifies resource isolators - memory will be increased by 128MB above summary value of particular applications.

What if one (for example from ten) application has specified memory isolator, and rest of them does not have such restrictions? Whole pod will be limited to these application limit + 128MB for system/rest? It looks quite bad.